<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerKeysSettings</class>
 <widget class="QWidget" name="PowerKeysSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Handle Power Keys</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="powerKeyLabel">
        <property name="text">
         <string>Power Key Action:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="powerKeyActionComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="suspendKeyActionComboBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="hibernateKeyActionComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="suspendKeyLabel">
        <property name="text">
         <string>Suspend Key Action:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="hibernateKeyLabel">
        <property name="text">
         <string>Hibernate Key Action:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="waylandLabel">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
